controllers/database_controller.go

```diff
@@ -117,6 +122,7 @@ func (r *DatabaseReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *DatabaseReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	r.recorder = mgr.GetEventRecorderFor("containerset-controller")
```

Comment:
"containerset-controller" -> "database-controller"

Reply:
"GetEventRecorderFor returns a new EventRecorder for the provided name"

Here it is just passed as name for the recorder and is not a keyword. I have fixed it. Will add a comment for more clarity